<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendsList</class>
 <widget class="QDialog" name="FriendsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:url(:/img/img/menuBg.png)</string>
  </property>
  <widget class="QTextBrowser" name="chatBrowser">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>391</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="OnlineBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RefreshButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Raffraichir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
